src/ingestors/zora-internal/zora-metadata.ts

```diff
@@ -61,14 +61,7 @@ export class ZoraMetadataProvider {
     if (fallbackStartDateValue) {
       startDate = new Date(parseInt(fallbackStartDateValue) * 1000);
     }
-
-    let fallbackEndDateValue =
```

Comment:
Maybe we get the end date and only if it's a non-0 value consider it?

Many Zora mints do have end dates that we probably want to capture if available?

Reply:
but do we have any evidence they're ever represented by the data on the sale strategies...?

Reply:
I believe they were at some point (these ingested properly until a week or so ago afaik)

I'm not sure whether it's a change in the data layout or the prevailing strategy in play though.